README.md

```diff
@@ -18,6 +18,13 @@ Or just a single theme `@sweetalert2/theme-<theme_name>`, e.g.
 npm install --save sweetalert2 @sweetalert2/theme-dark
 ```
 
+Or grab them both from [jsdelivr CDN](https://cdn.jsdelivr.net/npm/@sweetalert2/themes@latest/), for example:
```

Comment:
I believe this should be the link https://www.jsdelivr.com/package/npm/@sweetalert2/themes